ibm/data_source_ibm_cm_version.go

```diff
@@ -0,0 +1,209 @@
+			},
+			"target_version": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The semver value for this new version, if not found in the zip url package content.",
+			},
+			"repo_type": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The type of repository containing this version.  Valid values are 'public_git' or 'enterprise_git'.",
+			},
+			"rev": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Cloudant revision.",
+			},
+			"crn": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Version's CRN.",
+			},
+			"version": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Version of content type.",
+			},
+			"sha": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "hash of the content.",
+			},
+			"catalog_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Catalog ID.",
+			},
+			"kind_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Kind ID.",
+			},
+			"repo_url": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Content's repo URL.",
+			},
+			"source_url": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Content's source URL (e.g git repo).",
+			},
+			"tgz_url": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "File used to on-board this version.",
+			},
+			"single_instance": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "Denotes if single instance can be deployed to a given cluster.",
+			},
+			"image_manifest_url": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "If set, denotes a url to a YAML file with list of container images used by this version.",
+			},
+			"deprecated": &schema.Schema{
+				Type:        schema.TypeBool,
+				Computed:    true,
+				Description: "read only field, indicating if this version is deprecated.",
+			},
+			"package_version": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Version of the package used to create this version.",
+			},
+			"version_locator": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "A dotted value of `catalogID`.`versionID`.",
+			},
+			"console_url": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Console URL.",
+			},
+			"long_description": &schema.Schema{
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Long description for version.",
+			},
```

Comment:
Other response attributes like portal_approval_record, portal_ui_url,repo_info,provider.. are not supported in this datasource.. Any specific reason for that?